hack/verify-gendocs.sh

```diff
@@ -39,20 +39,25 @@ if [[ ! -x "$gendocs" || ! -x "$genman" || ! -x "$genbashcomp" ]]; then
 fi
 
 DOCROOT="${KUBE_ROOT}/docs/"
-TMP_DOCROOT="${KUBE_ROOT}/docs_tmp/"
+TMP_DOCROOT="${KUBE_ROOT}/_tmp/docs/"
 
+mkdir -p "${TMP_DOCROOT}"
 cp -a "${DOCROOT}" "${TMP_DOCROOT}"
 
 kube::util::gen-doc "${genman}" "${TMP_DOCROOT}/man/man1/"
 kube::util::gen-doc "${gendocs}" "${TMP_DOCROOT}"
-
 echo "diffing ${DOCROOT} against freshly generated docs"
 set +e
 diff -Naupr -I 'Auto generated by' "${DOCROOT}" "${TMP_DOCROOT}"
 ret=$?
 set -e
-rm -rf "${TMP_DOCROOT}"
-if [ $ret -eq 0 ]
+rm -rf $(dirname "${TMP_DOCROOT}")
```

Comment:
The result of `$(dirname ...)` should be double quoted, maybe save it in a temporary variable.